src/main/java/com/google/jenkins/plugins/computeengine/InstanceConfiguration.java

```diff
@@ -351,6 +350,7 @@ public ComputeEngineInstance provision() throws IOException {
     /** Initializes transient properties */
     protected Object readResolve() {
         labelSet = Label.parse(labels);
+        this.networkInterfaceIpStackMode = new NetworkInterfaceSingleStack(externalAddress);
```

Comment:
By default and for retrocompatibility we're creating a single stack network interface with the value of the deprecated `externalAddress` field.